css/style.css

```diff
@@ -140,12 +140,28 @@ hgroup.text {
 
 .download {
 	float: right;
-	width: 186px;
+	width: 150px;
+}
+
+.download .download-icon {
+	text-align: center;
+	margin: -18px 0 -6px;
 }
 
 .download a {
+	color: #757575;
 	display: inline-block;
 }
+.download a:hover {
+	color: #5490ca;
+}
+
+.download .mega-octicon {
+	font-size: 112px;
+    text-align: center;
+    text-shadow: 2px 2px 5px #666666;
```

Comment:
The actual site uses the minified version of the css which has the short form. For the dev version of the css, long form is fine.